server/service/mdm.go

```diff
@@ -1455,6 +1459,9 @@ func (svc *Service) BatchSetMDMProfiles(ctx context.Context, tmID *uint, tmName
 	if err != nil {
 		return ctxerr.Wrap(ctx, err, "getting app config")
 	}
+	if assumeEnabled {
+		appCfg.MDM.WindowsEnabledAndConfigured = true
```

Comment:
What fails if you don't do this?

Reply:
Dry run fails when I try to enable and configure MDM at the same time.